compute_tools/src/compute.rs

```diff
@@ -736,9 +754,27 @@ impl ComputeNode {
         // Log metrics so that we can search for slow operations in logs
         info!(?metrics, postmaster_pid = %postmaster_pid, "compute start finished");
 
+        let features = &pspec.spec.features;
+        if features.contains(&ComputeFeature::PrewarmLfcOnStartup) {
+            let cloned = self.clone();
+            rt.spawn(async move { cloned.prewarm_lfc().await });
```

Comment:
`crate::http::prewarm_lfc` now spawns a separate task. Do we need to spawn a new task here?

Reply:
Yes, as prewarm may take significant time, we don't want to block returning from this function

Reply:
You are right in the sense this task does nothing but invoke another async function which calls spawn, but we don't have other choice to call it.
We can call it with spawn_blocking but I don't see any advantages with that approach

Reply:
Fair. `start_compute` is sync and this outer task will exit after spawning the inner task.